driftcounter_s.lua

```diff
@@ -18,6 +25,53 @@ Citizen.CreateThread(function()
 	end)
 end)
 
+Citizen.CreateThread(function()	
+	RegisterNetEvent("RequestConfig")
+	AddEventHandler("RequestConfig", function()
+		TriggerClientEvent("RecieveConfig", source, SaveAtEndOfDrift, SaveTime)
+	end)
+	
+	if UseGlobalScore then
+		
+		RegisterNetEvent("SaveScore")
+		AddEventHandler("SaveScore", function(client, data)
+			UpdatePlayerInDB(client, data)
+		end)
+		
+		RegisterNetEvent("LoadScoreData")
+		AddEventHandler("LoadScoreData", function()
+			GetPlayerInfo(source)
+		end)
+
+		function GetPlayerInfo(client)
+			PerformHttpRequest('https://drift-counter-scores.firebaseio.com/scores.json', function(statusCode, text, headers)
+				if text then
+					local info = json.decode(text)
+					for _,users in pairs(info) do
+						if GetPlayerIdentifier(client, 0) == users.identifier then
+							TriggerClientEvent("LoadScore", client, users.score)
+						else
```

Comment:
wouldn't this just create a new user if the client wasn't found?

Reply:
Wouldn't it only create a new user if it cannot find an account with that identifier?

Reply:
i dont think so? since you check it for each iteration, if the current iteration wasn't the player, it would try to create a new one for every iteration that ISN'T the correct client

Reply:
Oh yeah, so it does, silly me